lexicons/com/atproto/label/label.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.label.label",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Metadata tag on an atproto resource (eg, repo or record)",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["src", "uri", "val", "ts"],
+        "properties": {
+          "src": {"type": "string", "maxLength": 128},
+          "uri": {"type": "string", "maxLength": 1024},
+          "cid": {"type": "string", "maxLength": 128},
+          "val": {"type": "string", "maxLength": 128},
+          "ts": {"type": "datetime"},
```

Comment:
we've been using `createdAt` for this.

i think it helps differentiate because there are several different "timestamps" in the system
- createdAt
- indexedAt
- (not to be used as a timestamp) but possibly the cursor on the event stream
- (again, not to be used as a timestamp), but the TID

Reply:
I agree that "timestamp" can mean lots of things, it is important to clarify which meaning, and the name of the field is a good way to do that. And to be consistent with other lexicons.

(as a side-reference, scholarly publishing APIs can turn in to rats nest of different "publication times" and "update times": https://github.com/CrossRef/rest-api-doc#sorting)

I went with `ts` here to be consistent with the proposed free-floating label schema, where I feel like the byte savings might be worth it in the context of many labels getting stapled to records all over the place. Also why the other fields are short. Maybe `cts` ("created timestamp") or `cat` ("created at")? The JWT 3-char field name is `iat` ("issued at").

Reply:
I definitely get the logic of the smaller labels but it's different than the naming conventions we've used with all the others, so if we change that up we should make sure it's with a specific intent